share/src/main/java/io/zensoft/share/config/Facebook/facebookConfig

```diff
@@ -0,0 +1,6 @@
+appId=1482536371851716
+appSecret=xxx
+pageId=179314156095163
+userId=103714360544349
```

Comment:
this is security sensitive info, please add `config.example` to git.  `config` file should be in gitignore

Reply:
Ok, will be corrected

Reply:
Is ID also a security sensitive info? It is easy to find facebook ID of any page or user